### ISSUE TITLE
If only the data layer has entities，so domain need depends on data. This breaks the dependency rules right?

### ISSUE THREAD

**Issue by annybudong** [android10/Android-CleanArchitecture#299]
If only the data layer has entities，so domain need depends on data. This breaks the dependency rules right?

**Comment by drayan85** [comment]
No, You have to use the mapper classes to convert the model classes to Entity and Entity to model class.
Please look at the `UserEntityDataMapper .java`

**Comment by annybudong** [comment]
> No, You have to use the mapper classes to convert the model classes to Entity and Entity to model class.
> Please look at the `UserEntityDataMapper.java`

thanks so much
